src/pages/Administrate.tsx

```diff
@@ -81,8 +83,9 @@ const Administrate = () => {
       )}
       {!txResult && (
         <Container>
-          <p>Contract transaction ID</p>
+          <label htmlFor="tx-id">Contract transaction ID</label>

```

Comment:
Good job with applying the best practices of testing-library and improving accessibility 🎖️